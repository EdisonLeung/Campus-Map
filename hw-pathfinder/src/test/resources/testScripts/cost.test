# Tests the total cost calculating functionality of pathfinding

# Create Graph
CreateGraph cost

# Add Nodes
AddNode cost a
AddNode cost b
AddNode cost c

# Add Edges
AddEdge cost a b 1.0
AddEdge cost b c 1.0

# Test Pathfinding
FindPath cost a c