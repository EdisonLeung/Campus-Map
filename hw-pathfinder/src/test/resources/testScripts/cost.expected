# Tests the total cost calculating functionality of pathfinding

# Create Graph
created graph cost

# Add Nodes
added node a to cost
added node b to cost
added node c to cost

# Add Edges
added edge 1.000 from a to b in cost
added edge 1.000 from b to c in cost

# Test Pathfinding
path from a to c:
a to b with weight 1.000
b to c with weight 1.000
total cost: 2.000